htdocs/mri/jiv/get_file.php

```diff
@@ -133,6 +141,12 @@
     $MimeType         = 'image/vnd.nrrd';
     $DownloadFilename = basename($File);
     break;
+case 'DICOMTAR':
+    // ADD case for DICOMTAR
+    $FullPath         = $tarchivePath . '/' . $File;
+    $MimeType         = 'application/x-tar';
+    $DownloadFilename = basename($File);
+    break;
```

Comment:
Can you split this into 2 PRs? The DICOM Tar download and the imaging browser linked instruments don't belong together.

Reply:
I will. I just wanted to see if there is interest in this. But I guess I will split them, basing it off of minor, and take care of your other comments in the newly sent pull requests to make it easier for review.